pentasharp/Controllers/BusSchedule.cs

```diff
@@ -63,28 +84,56 @@ public IActionResult GetRoutes()
         [HttpPut("EditRoute/{id}")]
         public IActionResult EditRoute(int id, [FromBody] AddRouteViewModel model, int hours, int minutes)
         {
-            var route = _context.BusRoutes.Find(id);
+            var route = _context.BusRoutes.SingleOrDefault(r => r.RouteId == id);
```

Comment:
if routeId is a primary key, i guess its better for us to stick with Find, if not keep this approach